TLM/TLM/Manager/Impl/LaneConnection/LaneConnectionSubManager.cs

```diff
@@ -462,7 +500,9 @@ public class LaneConnectionSubManager :
                 return;
             }
 
-            if (!HasOutgoingConnections(laneId, startNode)) {
+            const bool RESET = false; // reset lane arrows when last lane connection is removed.
+
+            if (RESET && !HasOutgoingConnections(laneId, startNode)) {
```

Comment:
I also dont get the point of this code. i understand it for #ifdef but whats the point here?

Reply:
cool I remove it.